### PR TITLE
Add Wikidata second labeling campaign

### PR DESCRIPTION
**Pull request by Ladsgroup**
Bug: T195701

### CODE REVIEW

File: Makefile.manual [hunk 1]
```diff
@@ -331,6 +331,31 @@ datasets/wikidatawiki.labeled_revisions.w_cache.20k_2015.json: \
...
+		datasets/wikidatawiki.sampled_revisions.500k_2018.json
+	cat $< | \
+	./utility autolabel --host=https://www.wikidata.org \
+		--trusted-groups=bot,sysop,bureaucrat \
+		--trusted-edits=1000 \
+		--revert-radius=3 \
+		--revert-window=48 \
+		--verbose > $@
+
+datasets/wikidatawiki.autolabeled_revisions.500k_2018.no_review.json: \
```
Comment (halfak) [inline Makefile.manual]: We don't need to split these up.  I think we just want to submit the observation that "need_review" to wikilabels.

File: Makefile.manual [hunk 2]
```diff
@@ -331,6 +331,31 @@ datasets/wikidatawiki.labeled_revisions.w_cache.20k_2015.json: \
...
+		datasets/wikidatawiki.autolabeled_revisions.500k_2018.json
+	cat $< | grep -E '"needs_review": (false|"False")' > $@
+
+datasets/wikidatawiki.autolabeled_revisions.500k_2018.review.json: \
+		datasets/wikidatawiki.autolabeled_revisions.500k_2018.json
+	cat $< | grep -E '"needs_review": (true|"True")' > $@
+
+datasets/wikidatawiki.revisions_for_review.5k_2018.json: \
+		 datasets/wikidatawiki.autolabeled_revisions.500k_2018.review.json
+	cat $< | shuf -r -n 5000 > $@
```
Comment (halfak) [inline Makefile.manual]: Are there substantially more than 5k?  Also the -r is certainly unwanted.  It means to shuffle with replacement, so if there are less than 5k, there will be duplicates.

Comment (Ladsgroup) [reply Makefile.manual]: They are 20K

File: Makefile.manual (hunk 1 above)
Comment (Ladsgroup) [reply Makefile.manual]: Done

File: Makefile.manual (hunk 2 above)
Comment (halfak) [reply Makefile.manual]: We should cut the initial sample down by a factor of 4 then to get to ~5k.  So 125k sampled revisions --> autolabel --> 5k needs review.  Does that make sense?

**Comment by Ladsgroup** [comment]
@halfak Done!

**Pull request merged**